rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -155,6 +164,40 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
             take=1,
             method='acquisition',
         )
+        swap_key = str(HistoryEventType.TRADE) + '__' + str(HistoryEventSubType.SPEND)
```

Comment:
why is this here? Don't all swaps already have their own rules in each accountant module? IF this is here then shouldn't those accountants be removed?